Fixed map frame being slightly offset when 'Movable World Map' is enabled

BlacRyu opened the pull request:
This fixes the windowed world map frame position so that it fits within the border.

I've only tested this on Epoch so I don't know if this was an Epoch-only bug.

Before: 
<img width="1005" height="744" alt="image" src="https://github.com/user-attachments/assets/c2d0b163-f4f9-4d62-a915-5f6114be57a5" />




After: 
<img width="995" height="735" alt="image" src="https://github.com/user-attachments/assets/6a59ccab-d4e0-42fe-81c8-9488b2177cb2" />

BlacRyu committed:
- Fixed map frame being slightly offset in windowed mode

rissole commented:
<img width="1638" height="1145" alt="image" src="https://github.com/user-attachments/assets/799d2d51-3d61-4c61-9f1f-50bade5524e5" />

Thank you for the contribution, I've tested on epoch too but it's not aligned 🤔  (no addons enabled except Magnify on this branch)

I think this needs a bit more investigation - this is definitely an issue (I've seen other reports) but I'm not sure if it's just me or some other per-character setting that isn't being cleared properly

I'm going overseas for a month tomorrow, so I won't be able to test out or investigate further 🙁 If you happen to find/explain the root cause + bump the version, I'll be happily to hit Merge PR while I'm away though 🙂

BlacRyu commented:
Yeah, strange. I tested with no mods as well. If I figure out anything I'll let you know. I've just been playing a Night Elf Druid so far and it's always been broken like that for me.

It doesn't look like tweaking display resolution or UI scale have any effect on it on my end.

BlacRyu commented:
Looks like it's caused by the `Movable World Map` option under Interface -> Objectives. I'll look into it more later and see if I can get a solution that fixes it in both cases.

<img width="778" height="638" alt="image" src="https://github.com/user-attachments/assets/7ca4c3f9-ddeb-4ac3-964a-4c4cb2819030" />

BlacRyu committed:
Fix the frame whether 'Movable World Map' is enabled or not

- Sets the WorldMapScrollFrame anchor point based on whether WORLDMAP_SETTINGS.advanced is enabled or not (which is the 'Movable World Map' option). Uses the same values as what the default UI sets on WorldMapDetailFrame in WorldMapFrame_SetMiniMode() - https://github.com/wowgaming/3.3.5-interface-files/blob/main/WorldMapFrame.lua#L2005

BlacRyu committed:
Bump version to 1.4.3

BlacRyu commented:
Okay I fixed it in both instances now, seems to work fine as far as I can tell. I bumped the version to 1.4.3

I'm not sure how the WORLDMAP_QUESTLIST_SIZE feature works. I don't know if that mode is normally available and just disabled on Epoch or how that works, so I can't test it for now. Should be unaffected though.

BlacRyu commented on Main.lua:
```diff
@@ -213,20 +213,18 @@ function Magnify.SetupWorldMapFrame()
         WorldMapFrame:SetPoint("TOPLEFT", WorldMapScreenAnchor, 0, 0);
         WorldMapFrame:SetScale(WorldMapScreenAnchor.preferredMinimodeScale);
         WorldMapFrame:SetMovable("true");
-        WorldMapTitleButton:Show()
-        WorldMapTitleButton:ClearAllPoints()
-        WorldMapTitleButton:SetPoint("TOPLEFT", WorldMapFrame, "TOPLEFT", 13, -13)
-        WorldMapFrameTitle:Show()
-        WorldMapFrameTitle:ClearAllPoints();
-        WorldMapFrameTitle:SetPoint("CENTER", WorldMapTitleButton, "CENTER", 32, 0)
```
Also, I removed these lines setting the anchor points for the WorldMapTitleButton and WorldMapFrameTitle as they seem to be positioned properly natively. If this was needed for some reason I'm unaware of, let me know and I can restore it.

BlacRyu replied:
Nvm, it was needed to keep the text in place when panning the map around. Added it back and fixed it to be positioned correctly in both modes.

BlacRyu committed:
The title text no longer moves around with the map

BlacRyu committed:
Merge branch 'main' into fix-frame-position
